### PR TITLE
Adding '/usr/bin/env bash'

### PR DESCRIPTION
**Pull request by truongnh1992**
This commit aims to add '/usr/bin/env bash' as a shebang line
to indicates scripts use bash shell for interpreting.

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/update-generated-kubelet-plugin-registration-dockerized.sh
Comment (BenTheElder) [inline hack/update-generated-kubelet-plugin-registration-dockerized.sh]: ./shrug
this runs inside a container anyhow

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve

**Comment by truongnh1992** [comment]
> /lgtm
> /approve

Thanks for your review.

**Comment by truongnh1992** [comment]
> /lgtm
> /approve

I wonder why the PR doesn't get **approve label** after you commented `/approve`?

**Comment by BenTheElder** [comment]
@truongnh1992 the comment above https://github.com/kubernetes/kubernetes/pull/74107#issuecomment-464234716 explains, I'm a reviewer in those directories but not an approver currently.

/assign @dims

**Comment by truongnh1992** [comment]
> 
> 
> @truongnh1992 the comment above [#74107 (comment)](https://github.com/kubernetes/kubernetes/pull/74107#issuecomment-464234716) explains, I'm a reviewer in those directories but not an approver currently.
> 
> /assign @dims

Thank you. I got it.

**Pull request merged**
